src/zircon/frontend/roundrobin.go

```diff
@@ -0,0 +1,44 @@
+package frontend
+
+import (
+	"zircon/apis"
+	"sync"
+)
+
+type roundrobin struct {
+	servers []apis.Frontend
+
+	mu     sync.Mutex
+	nextID int
+}
+
+// Constructs an interface to a set of frontends as if they were one front-end.
+// Sends each request to another server. TODO: this is probably a bad way to do it!
```

Comment:
Eh, this is easy enough to fix to do better in the future

Reply:
Yup.